sdks/python/apache_beam/typehints/native_type_compatibility_test.py

```diff
@@ -49,6 +49,11 @@ class _TestGeneric(typing.Generic[T]):
   pass
 
 
+class _TestPair(typing.NamedTuple('DataPair', [('base', T), ('diff', T)]),
```

Comment:
please select some generic values for the stub to unnecessary association with domain-specific code where the snippet came from.

Reply:
done